girder/utility/filesystem_assetstore_adapter.py

```diff
@@ -70,6 +70,20 @@ def validateInfo(doc):
             raise ValidationException(
                 'Unable to write into directory "%s".' % doc['root'])
 
+        if doc.get('perms') is None:
```

Comment:
This should just be `if not doc.get('perms'):` so that if the user leaves that field blank they won't get errors.